nilearn/plotting/displays/_axes.py

```diff
@@ -414,11 +414,9 @@ def transform_to_2d(self, data, affine):
             The affine of the volume.
 
         """
-        if self.direction in "xlr":
-            max_axis = 0
-        else:
-            max_axis = ".yz".index(self.direction)
-
+        max_axis = (
+            0 if self.direction in "xlr" else ".yz".index(self.direction)
+        )
         # set unselected brain hemisphere activations to 0
```

Comment:
```suggestion
        # set unselected brain hemisphere activations to 0
        # kwargs should have priority over this_kwargs
        # so that the user can override the default logic        
```